src/Etats/play.ts

```diff
@@ -23,8 +23,7 @@ let s_invers = 1 / s;
 let fps_adapt = 1;
 const fps_array = [];
 
-// Animation de fin et de début
-let anim = true;
+let levelStarted = false;
```

Comment:
Maybe `isLevelPlaying` as it also seems to be `false` on the end of the level 🤷‍♂ ?

Reply:
Agree it is a better name ;)